test/functional/wallet_resendwallettransactions.py

```diff
@@ -49,16 +49,21 @@ def run_test(self):
         block.solve()
         node.submitblock(ToHex(block))
 
-        # Transaction should not be rebroadcast
         node.syncwithvalidationinterfacequeue()
-        node.p2ps[1].sync_with_ping()
-        assert_equal(node.p2ps[1].tx_invs_received[txid], 0)
+        now = int(time.time())
+
+        # Transaction should not be rebroadcast within first 12 hours
+        # Leave 2 mins for buffer
+        twelve_hrs = 12 * 60 * 60
+        two_min = 2 * 60
```

Comment:
I'm curious - any reason why we chose 2 minutes here?

Reply:
It can probably be reduced to one second, if `now` gets set to `block_time` (the mock time before this call)